### ISSUE TITLE
Error during zsh initialization:   _arguments:comparguments:325: can only be called from completion function

### ISSUE THREAD

**Issue by danylkaaa** [lets-cli/lets-zsh-plugin#1]
I use oh-my-zsh + powerlevel10k and I always catch this output in my console during zsh initialization:
```sh
_arguments:comparguments:325: can only be called from completion function
```

My Config:
```sh
if [[ -r "${XDG_CACHE_HOME:-$HOME/.cache}/p10k-instant-prompt-${(%):-%n}.zsh" ]]; then
  source "${XDG_CACHE_HOME:-$HOME/.cache}/p10k-instant-prompt-${(%):-%n}.zsh"
fi

export ZSH=~/.oh-my-zsh
export UPDATE_ZSH_DAYS=5
ZSH_THEME="powerlevel10k/powerlevel10k"

plugins=(
  git
  npm
  lets
  colored-man-pages
  iterm2
  zsh-autosuggestions
  docker-compose
  zsh-syntax-highlighting
)

source ~/.oh-my-zsh/custom/plugins/zsh-autosuggestions/zsh-autosuggestions.zsh 

autoload -U compinit && compinit
```

**Comment by kindermax** [comment]
Yep, its a bug. Will fix it soon

**Comment by kindermax** [comment]
Fixed in `lets` `0.0.17` version. Check out new (update) documentation on how to properly install plugin
